STYLIZING.md

```diff
@@ -0,0 +1,104 @@
+# Stylizing
+
+You can configure the look and feel of your prompt easily with some built-in
+options. This includes changing foreground and background colors, replacing
+icons and separators, adding newlines...
+**Please be aware that styling options should always be set before loading/sourcing P9K!
+Not every option can be changed at runtime.**
+
+## General Stylizing Options
+
+|Option|Default|Effect|
+|------|-------|------|
+|`P9K_PROMPT_ON_NEWLINE`|`false`|Display the prompt on the next line. Also checkout the sgement `newline`.|
+|`P9K_RPROMPT_ON_NEWLINE`|`false`|Display `RPROMPT` on the same line as `PROMPT` if `P9K_PROMPT_ON_NEWLINE=true`|
+|`P9K_MULTILINE_FIRST_PROMPT_PREFIX_ICON`|`?`|If the prompt is more than one line, this will be the prefix for the first line.|
+|`P9K_MULTILINE_NEWLINE_PROMPT_PREFIX_ICON`|`?`|If the prompt is at least tree lines, this will be the prefix for all lines that are neither the first nor the last.|
+|`P9K_MULTILINE_LAST_PROMPT_PREFIX_ICON`|`?`|If the prompt is more than one line, this will be the prefix for the last line.|
+|`P9K_PROMPT_ADD_NEWLINE`|`false`|Adds a newline before displaying the prompt|
+|`P9K_PROMPT_ADD_NEWLINE_COUNT`|`1`|Change the amound of newlines added if `P9K_PROMPT_ADD_NEWLINE=true`.|
+|`P9K_DISABLE_RPROMPT`|`false`|Disables `RPROMPT`.|
+|`P9K_COLOR_SCHEME`|unset|Set to `light` to invert default background and foreground color.|
+|`P9K_{LEFT,RIGHT}_SEGMENT_SEPARATOR_ICON`|`?`|This is the separator between two segments of different color.|
```

Comment:
This is a special case, as these characters are not included in normal fonts.. Not sure if we want to display an image instead..

Reply:
I replaced the question marks with the separator spaces for now but I think you are right. Not everybody will see it correctly. I'm not sure where to put the pictures to though...

Reply:
I always put the images in the github cdn. Just drag and drop your images in a reply textbox and you'll get a link like `https://cloud.githubusercontent.com/assets/`..
This solution may not be optimal, maybe we should put the images in the repo as well someday.

Reply:
// Comment to keep the path to the images permanent
![image](https://user-images.githubusercontent.com/16988672/62417083-99285000-b647-11e9-9b3f-88f8e9bae963.png)